solutions/cf-1560B.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: cf-1560B
+source: CF
+title: Who's Opposite?
+author: Ryan Chou
+---
+
+[Official Editorial (C++)](https://codeforces.com/blog/entry/94009)
+
+## Explanation
+
+Since the number of people left of $a$ and the number of people right of $b$ must be equal, we can find the total number of people in the circle with $abs(a - b) * 2$. 
+
+From here, we can see if this circle is possible with $a, b, c > numGroup$, 
+
+If any of these exceed the number of people in the circle, they can't have a partner. 
+
+Otherwise, we can subtract or add $abs(a - b)$ from $c$ to find the answer.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(1)$
+
+<LanguageSection>
+
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <stdlib.h>
```

Comment:
what's this for now

Reply:
iostream is for input/output and
stdlib.h is for abs and stuff

Reply:
thought abs was alr defined or smth?

Reply:
oops yeah you're right